utils/logger.js

```diff
@@ -38,35 +38,19 @@ function formatMessage(message, meta) {
   return message;
 }
 
-module.exports.log =
-  (message, meta) => {
-    if (logdnaLogger) {
-      logdnaLogger.log(formatMessage(message, meta), { meta });
-    }
-    bunyanLogger.info(message);
-  };
-
-module.exports.debug =
-  (message, meta) => {
+function commonLogger() {
+  return (message, meta) => {
     if (logdnaLogger) {
       logdnaLogger.debug(formatMessage(message, meta), { meta });
```

Comment:
I don't like the fact that no matter which function you call, it'll always end up executing `logdnaLogger.debug` and `bunyanLogger.debug`.
Could you make `commonLogger` consider the log level parameter and call the corresponding functions on bunyan and logdna?